### ISSUE TITLE
SSL Support for proxied requets.

### ISSUE THREAD

**Issue by keremturhan1992** [shekyan/slowhttptest#91]
Hi there,

The current parameter "-d" with proxy does not work on https sites and it forces to use HTTP protocol.

**Comment by shekyan** [comment]
You are right, the reason is that the old tool never sends CONNECT - it only issues an absolute-form request to the proxy, which can't carry TLS. So -d against an https target was never going to work.

slowhttptest-ng does CONNECT tunnelling with TLS inside the tunnel, so -d works against https there. There's also -e if you want only the availability probe going through the proxy and the attack direct.

Not planning to backport this to the old tool.
